jme3-examples/src/main/java/jme3test/light/pbr/RefEnv.java

```diff
@@ -161,5 +162,9 @@ public void done(LightProbe result) {
             rootNode.addLight(probe);
 
         }
+
+        if (eCam.isBusy()) {
+            System.out.println("EnvironmentCamera busy as of frame " + frame);
```

Comment:
Question: can it be possible that logging is done here every frame? Because that can slow things down.

Reply:
Nope, based on https://github.com/jMonkeyEngine/jmonkeyengine/issues/1446#issuecomment-806053365 it will be busy just for one frame!